Dockerfile

```diff
@@ -1,10 +1,11 @@
 FROM alpine:latest
 RUN apk --update add ca-certificates \
                      mailcap \
-                     curl
+                     curl \
+                     jq
 
 HEALTHCHECK --start-period=2s --interval=5s --timeout=3s \
-  CMD curl -f http://localhost/health || exit 1
+  CMD curl -f http://localhost:$(jq '.port' /.filebrowser.json || 80)/health || curl -f http://localhost/health || exit 1
```

Comment:
`|| curl -f http://localhost/health` is redundant

Reply:
1. The default port is 80 with your docker config: https://github.com/filebrowser/filebrowser/blob/master/.docker.json
2. If the `.filebrowser.json` does not contain `"port" : 1234`, the `jq` command will fail, and no port will be returned. That's why we need a default function call.

Reply:
You already do it in this part `curl -f http://localhost:$(jq '.port' /.filebrowser.json || 80)/health`

Reply:
Indeed, the `|| 80`  was useless, as the command returns a `null` string if the `.port` value is not found. I've removed it, but we have to keep the fallback method in order to deal with json files without `port` value.

imho, you should definitely choose a port > 1023, and remove the `port` option from the json file. There is no reason to allow to change the port inside the container, as we can bind any host port to a container's one, and we can choose any container port in a proxy config (Traefik, ...).

The only problem I can see is the BC break for all people using the existing image with the port `80`